metadata/containers.go

```diff
@@ -61,7 +61,64 @@ func (s *containerStore) Get(ctx context.Context, id string) (containers.Contain
 		return containers.Container{}, errors.Wrapf(err, "failed to read container %q", id)
 	}
 
-	return container, nil
+	if len(fieldmasks) == 0 {
+		fieldmasks = []string{"id", "labels", "image", "runtime", "spec", "snapshotter", "snapshot_key", "created_at", "updated_at", "extensions"}
```

Comment:
This looks fragile, any reason not to just return or copy the entire container?

Reply:
Just wanted to keep similar logic to Update. Fixed this by returning `container`